lib/couchbaseX.js

```diff
@@ -147,8 +154,7 @@ class Couchbase extends NoSQL {
     // With some defaults.
     let query = ViewQuery.from(designDoc, viewName)
       .on_error(ViewQuery.ErrorMode.STOP)
-      .order(ViewQuery.Order.ASCENDING)
-      .stale(ViewQuery.Update.BEFORE);
```

Comment:
Good to remove the Update.BEFORE, but it's better to explicitly set the `Update.NONE` instead of relying on the default of SDK.